api/chat_main_handler.go

```diff
@@ -262,20 +217,7 @@ func (h *ChatHandler) OpenAIChatCompletionAPIWithStreamHandler(w http.ResponseWr
 		if shouldReturn {
 			return
 		}
-		// insert ChatMessage into database
-		chatMessageParams := sqlc_queries.CreateChatMessageParams{
-			ChatSessionUuid: chatSessionUuid,
-			Uuid:            answerID,
-			Role:            "assistant",
-			Content:         answerText,
-			UserID:          int32(userIDInt),
-			CreatedBy:       int32(userIDInt),
-			UpdatedBy:       int32(userIDInt),
-			Raw:             json.RawMessage([]byte("{}")),
-		}
-		log.Println(chatMessageParams)
-
-		m, err := h.chatService.q.CreateChatMessage(ctx, chatMessageParams)
+		m, err := h.chatService.CreateChatMessageSimple(ctx, chatSessionUuid, answerID, "assistant", answerText, userID)
 
 		log.Println(m)
 		if err != nil {
```

Comment:
:robot: OpenAI

This patch changes the `OpenAIChatCompletionAPIWithStreamHandler` function in `api/chat_main_handler.go` to use the `CreateChatMessageSimple` method of `h.chatService`.

This new method is not defined in this file, so it must be defined elsewhere. If it's not defined properly, this change could cause issues.

Please provide more context on where `CreateChatMessageSimple` is defined and how it works.

<!-- This is an auto-generated comment by OpenAI -->